cdap-ui/app/cdap/components/Footer/index.js

```diff
@@ -16,19 +16,24 @@
 
 import React from 'react';
 import { Theme } from 'services/ThemeHelper';
-
+import If from '../If';
 require('./Footer.scss');
 
 export default function Footer() {
   const footerText = Theme.footerText;
   const footerUrl = Theme.footerLink;
+  // 'project-id-30-characters-name1/instance-id-30-characters-name';
```

Comment:
What is this comment?

Reply:
Sample format of how it looks with Max 30 characters